etc/sib-config/sib-config.yml

```diff
@@ -59,6 +59,32 @@ sib_networks:
   - "{{ sib_network_internal }}"
   - "{{ sib_network_demo_vlan }}"
   - "{{ sib_network_demo_vxlan }}"
+  - "{{ sib_network_public }}"
+
+# Public network name.
+sib_network_public_name: "public"
+
+# Public network.
+sib_network_public:
+  name: "{{ sib_network_public_name }}"
+  provider_network_type: "vlan"
+  provider_physical_network: "physnet1"
+  provider_segmentation_id: 79
+  shared: True
+  # Subnet configuration.
+  subnets:
+    - "{{ sib_subnet_public }}"
+
+# Public subnet.
+sib_subnet_public:
+  name: "{{ sib_network_public_name }}"
+  cidr: "10.12.9.0/24"
+  gateway_ip: "10.12.9.1"
+  allocation_pool_start: "10.12.9.10"
```

Comment:
Can you run from 100-199 here, that doesn't leave much wriggle room, and I like nice round numbers...